index.html

```diff
@@ -198,12 +198,13 @@ <h1>contact me</h1>
       <p>If you have an application you are interested
          in developing, a feature that you need built or a project
           that needs coding. I’d
-         love to help with it </p>
-         <form class="form-start" action="">
+         love to help with it</p>
+         <form class="form-start"  action="https://formspree.io/f/xdovnkay"
+         method="POST">
           <input type="name" type="text" name="name" class="name" placeholder="write your name">
           <input type="email" type="email" name="email" class="email" placeholder="write your email">
-          <input type="message" type="text" name="message" class="message" placeholder="write your message">
-          <button class="start">get in touch</button>
+          <textarea type="message" type="text" name="message" class="message" placeholder="write your message"></textarea>
+          <button class="start" type="submit">Get in touch</button>
          </form>
```

Comment:
- [ ] - [ OPTIONAL ] We have some optional suggestions here, it is about your input fields, it is said that all of them should have these attributes in the image below, I would strongly recommend you add them as it is contributing to your project security and best HTML practices.

<img width="623" alt="1" src="https://user-images.githubusercontent.com/77843050/208181537-b8792aa5-762a-4152-9c87-2fb0c6dc0cad.png">

- [ ] - [ OPTIONAL ]You have added a pull request summary but it is not descriptive at all, please next time take a look back to some of our 1st sessions on how to create a pull request summary.

- [ ] This review comment [here](https://github.com/godybuoy/Portfolio-setup-and-mobile-first-/pull/3#discussion_r1050079067) mentioned that your mobile version style is broken and you would have to fix it, and am having the same broken issue again, please consider making all the necessary changes before you resubmit your work again, please take a look at this record below

![5](https://user-images.githubusercontent.com/77843050/208181981-d3578f02-aa8e-4053-b931-ce54309bd980.gif)